src/main/java/com/entropyteam/entropay/employees/clients/controllers/ClientController.java

```diff
@@ -0,0 +1,73 @@
+package com.entropyteam.entropay.employees.clients.controllers;
+
+import java.util.List;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Sort.Direction;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.ResponseEntity.BodyBuilder;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import com.entropyteam.entropay.employees.clients.dtos.ClientDto;
+import com.entropyteam.entropay.employees.clients.dtos.ClientSaveRequestDto;
+import com.entropyteam.entropay.employees.clients.dtos.ClientSaveResponseDto;
+import com.entropyteam.entropay.employees.clients.services.ClientService;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@RestController
+@RequestMapping(value = "/client", produces = MediaType.APPLICATION_JSON_VALUE)
+@Log4j2
+@RequiredArgsConstructor
+public class ClientController {
+
+    private final ClientService clientService;
+
+    @GetMapping
+    public ResponseEntity<Page<ClientDto>> findClients(
+            @RequestParam(defaultValue = "0") int page,
```

Comment:
I think we should use an index 1 for pages.

Reply:
Nop, it starts with 0, I've tested it.

Reply:
Test payload page 1: 
```
{
    "content": [
        {
            "id": "1ee1b9c0-8e5c-4abe-84ab-43b7bab51246",
            "name": "Leo test Client 7d09cbb4-c597-494c-bfe1-7d329e1b87b4",
            "address": "Bauness 2222 CABA",
            "zipCode": "1431",
            "city": "CABA",
            "state": "CABA",
            "country": "Argentina",
            "contact": "111111111",
            "preferredCurrency": "USD",
            "createdOn": "2022-06-03T11:00:54.031122",
            "modifiedOn": "2022-06-03T11:00:54.031135"
        },
        {
            "id": "069c4340-4fbc-42f8-be5e-e66f957ddb93",
            "name": "Leo test Client 9e5fc98d-bc39-49f9-9321-426a5d310a49",
            "address": "Bauness 2222 CABA",
            "zipCode": "1431",
            "city": "CABA",
            "state": "CABA",
            "country": "Argentina",
            "contact": "111111111",
            "preferredCurrency": "USD",
            "createdOn": "2022-06-03T11:00:54.383427",
            "modifiedOn": "2022-06-03T11:00:54.383442"
        },
        {
            "id": "1684b494-a635-426a-927a-deb999a634b5",
            "name": "Leo test Client b0f955a6-2aed-487a-90a0-23fb28836889",
            "address": "Bauness 2222 CABA",
            "zipCode": "1431",
            "city": "CABA",
            "state": "CABA",
            "country": "Argentina",
            "contact": "111111111",
            "preferredCurrency": "USD",
            "createdOn": "2022-06-03T11:00:53.691427",
            "modifiedOn": "2022-06-03T11:00:53.691452"
        },
        {
            "id": "cdb69856-7edc-48cb-8624-aabd11ff9a9f",
            "name": "Leo test Client c0cf2799-023e-4283-8549-917973572481",
            "address": "Bauness 2222 CABA",
            "zipCode": "1431",
            "city": "CABA",
            "state": "CABA",
            "country": "Argentina",
            "contact": "111111111",
            "preferredCurrency": "USD",
            "createdOn": "2022-06-03T11:00:55.079084",
            "modifiedOn": "2022-06-03T11:00:55.079097"
        },
        {
            "id": "89dc0a13-72a7-40ea-b6ca-877414bcd80b",
            "name": "Leo test Client ec5622ff-8a88-4a6e-b365-a3c54c573609",
            "address": "Bauness 2222 CABA",
            "zipCode": "1431",
            "city": "CABA",
            "state": "CABA",
            "country": "Argentina",
            "contact": "111111111",
            "preferredCurrency": "USD",
            "createdOn": "2022-06-03T11:00:15.050175",
            "modifiedOn": "2022-06-03T11:00:15.050198"
        },
        {
            "id": "2bdda9ff-1748-401d-97fa-fb7f138b1ca2",
            "name": "Leo test Client ee1bcf77-d987-4f0e-b827-474c73aeb5c4",
            "address": "Bauness 2222 CABA",
            "zipCode": "1431",
            "city": "CABA",
            "state": "CABA",
            "country": "Argentina",
            "contact": "111111111",
            "preferredCurrency": "USD",
            "createdOn": "2022-06-03T11:00:54.755212",
            "modifiedOn": "2022-06-03T11:00:54.755231"
        }
    ],
    "pageable": {
        "sort": {
            "empty": false,
            "sorted": true,
            "unsorted": false
        },
        "offset": 10,
        "pageNumber": 1,
        "pageSize": 10,
        "paged": true,
        "unpaged": false
    },
    "last": true,
    "totalPages": 2,
    "totalElements": 16,
    "size": 10,
    "number": 1,
    "sort": {
        "empty": false,
        "sorted": true,
        "unsorted": false
    },
    "first": false,
    "numberOfElements": 6,
    "empty": false
}
```

![image](https://user-images.githubusercontent.com/13138288/172628588-1580f23b-4176-4cb9-8720-283d62851975.png)

Reply:
@umarcostuc or do you mean to substract 1 to the param being passed to Pageable?